packages/ruby/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    readme-metrics (0.2.0)
+    readme-metrics (1.0.0)
```

Comment:
this is what is defined in the `version.rb` so after a `bundle install` this automatically updated itself